spec/controllers/users/registrations_controller_spec.rb

```diff
@@ -285,34 +285,26 @@
   end
 
   describe 'PUT #update' do
-    let!(:user) { create(:user) }
-    let(:params) { { id: '1982738917269816' } }
-    # let(:real_params) { { "user"=>{"name"=>"something", "about"=>"somethi", "skill_list"=>["Analytics"], 
-    #   "level_of_availability"=>"1-2 hours a day", "profile_links"=>"s", "location"=>"someth", 
-    #   "visibility"=>"0", "pair_with_projects"=>"0", "email"=>"cat@gmail.com", 
-    #   "password"=>"", "password_confirmation"=>"", "current_password"=>""} } }
+    let!(:user) { create(:user_complete_profile) }
+    let(:params) { { user: { id: user.id, email: 'changed@gmail.com' } } }
 
     before do
       sign_in user
+      session[:return_to] = projects_path
     end
 
     it 'we should be on projects path' do
       put :update, params: params
+      
+      user.reload
```

Comment:
This test wasn't passing because we weren't reloading the object before asserting in our test. I think you mentioned this but I said we don't reload anywhere else in our test. This is because we don't test the `update` action properly in the other controller specs - something we could correct!